jest.config.js

```diff
@@ -7,4 +7,7 @@ module.exports = {
     ...config.setupFiles,
     path.join(__dirname, './test/jest/jest.setup.js'),
   ],
+  transformIgnorePatterns: [
+    '/node_modules/(?!@k-int/stripes-kint-components|@folio/.*|ky/.*|uuid/.*)',
+  ],
```

Comment:
Isn't the pattern from `config` enough? This line everwrites it; when the config from `@folio/jest-config-stripes` is updated, we will not get those updates.

Reply:
Without it the tests for the **routes** and **src/edit/holdings/HoldingsForm.test.js** and **src/edit/items/ItemForm.test.js** are failing with:

```
/ui-inventory/node_modules/@k-int/stripes-kint-components/src/index.js:2
export * from './lib/hooks';
^^^^^^

SyntaxError: Unexpected token 'export'

   5 | import noop from 'lodash/noop';
   6 |
>  7 | import { NumberGeneratorModalButton } from '@folio/service-interaction';
```

Reply:
It is necessary to extend the existing config instead of replacing it.

For example:
```
const path = require('path');
const config = require('@folio/jest-config-stripes');

const additionalModules = ['@k-int/stripes-kint-components'].join('|');
const combinedModules = config.transformIgnorePatterns[0].replace(')', `|${additionalModules})`);

module.exports = {
  ...config,
  setupFiles: [
    ...config.setupFiles,
    path.join(__dirname, './test/jest/jest.setup.js'),
  ],
  transformIgnorePatterns: [combinedModules],
};
```

or mock the `@k-int/stripes-kint-components` in `__mock__` folder:

```
jest.mock('@folio/service-interaction', () => ({
  NumberGeneratorModalButton: jest.fn(() => <div>NumberGeneratorModalButton</div>),
}))
```

Reply:
Thanks! I fixed the `jest.config.js`